### ISSUE TITLE
Surface#savebmp may be broken

### ISSUE THREAD

**Issue by jacius** [rubygame/rubygame#2]
mstk reported this error in IRC:

```
surface.rb:604:in 'savebmp': undefined method 'SaveBMP' for SDL:Module (NoMethodError)
```

Probably a bug in Ruby-SDL-FFI.

[original LH ticket](http://jacius.lighthouseapp.com/projects/12816/tickets/54)

 This ticket has 0 attachment(s).

**Comment by jacius** [comment]
Filed rubygame/ruby-sdl-ffi#2 about the SaveBMP and LoadBMP functions missing from Ruby-SDL-FFI. I'll leave this issue open until that one is closed.

**Comment by jacius** [comment]
Fixed, see rubygame/ruby-sdl-ffi#3.
